### ISSUE TITLE
"An unhandled exception occurred: Sequence contains no matching element" while exporting a basic newly created project.

### ISSUE THREAD

**Issue by anand-010** [OpenKNX/Kaenx-Creator#42]
**Describe the bug**
I am first time trying to use this software I've created a sample project and resolved all errors but then I've tried to export my project i've got error
**An unhandled exception occurred: Sequence contains no matching element**
I've saw a similar issue  while importing a project #21  in my case it's a fresh project.
[saved project file ](https://drive.google.com/file/d/1uArU5ACNvESt7PfsG2p-TmDDnn8_P3uj/view?usp=sharing)

**Screenshots**
![KNX](https://github.com/user-attachments/assets/01d41040-f546-4a9f-86a6-e3a3198866b1)

**Desktop (please complete the following information):**
 - OS: win 10
 - Version: v1.8.0

**Comment by thewhobox** [comment]
Hallo anand,

du hast in deiner knxprod im allgemeinen Teil die Sprache Englisch, aber in der Applikation selbst Deutsch.
Das birgt Probleme, die bisher noch nicht bemerkt wurden.

Als Workaround: Sprache unter Allgemein auf Deutsch stellen.

**Comment by anand-010** [comment]
Thank you @thewhobox i've changed the language in in the application page and it exported.
but then i tried to import it into ets 5.7 i've got this error.
"The product has a different hash than the existing product."
![ets_err](https://github.com/user-attachments/assets/a97e441c-4fff-4f2f-8511-f0103d2f36b7)
i've previously imported other knxprod.
[output.zip](https://github.com/user-attachments/files/16479513/output.zip)

**Comment by anand-010** [comment]
i've solved it.
Under Settings -> Troubleshooting -> Clear product store memory and execute, then you could import the product data base in the project

**Comment by thewhobox** [comment]
You can only import an Application version once.
Even if you delete it from the catalog it is still in your projectfile.

But  i See you figured it out by yourself :)

**Comment by anand-010** [comment]
I've tried to create an "empty" project instead of choosing the "minimal ".
it created us English language as default in every parameter and objects I've only used English.
I've added necessary info about product and copied load procedures from a previous project.
but then I've verified the project I've got the same error "An unhandled exception occurred: Sequence contains no matching element".
so far i can only create project with the "minimal" option and Deutch language.
[PROJECT.zip](https://github.com/user-attachments/files/16645570/PROJECT.zip)
how do I solve it?
specific error message is more helpful.
